### DISCUSSION TITLE
Timezone how to set

### DISCUSSION THREAD

**Discussion by famousuniverse** [sidorares/node-mysql2#1448] [Q&A]
please advise on how to set the timezone, thanks

**Comment by sidorares** [comment]
@famousuniverse have you tried setting `timezone` connection option? 

from `mysqljs/mysql`  [documentation](https://github.com/mysqljs/mysql#connection-options)
> Can be 'local', 'Z', or an offset in the form +HH:MM or -HH:MM. (Default: 'local')

not sure if we are 100% with that but worth trying
